script.js

```diff
@@ -6,7 +6,140 @@ hamburger.addEventListener('click', () => {
+  portfolio.innerHTML += `
+    <div class="hero">
+    <div class="col1">
+      <img class="img-hero" src="${projects[i].image}" alt="image-project" />
+    </div>
+    <div class="col2">
+      <h2 class="title2">${projects[i].title}</h2>
+      <div class="type">
+        <span class="texttype">CANOPY</span>
+        <img src="images/dot.png" alt="dot" />
+        <span class="texttype2">Back End Dev</span>
+        <img src="images/dot.png" alt="dot" />
+        <span class="texttype2">2015</span>
+      </div>
+      <p class="text1">
+      ${projects[i].description}
+      </p>
+      <div class="wbtn">
+        <ul class="wbtn1">
+          <li class="wbtn3">${projects[i].html}</li>
+          <li class="wbtn3">${projects[i].css}</li>
+          <li class="wbtn3">${projects[i].javascript}</li>
+        </ul>
+      </div>
+      <button class="btn">See project</button>
+    </div>
+  </div>
+  `;
+}
+
+function displayProject(project) {
+  const modal = document.createElement('div');
+  modal.classList.add('modal');
+
+  modal.innerHTML = `
+    <div class="modal-content">
+      <span class="close">&times;</span>
+      <h2 class="h2-modal">${project.title}</h2>
+      <div class="type type-modal">
+        <span class="texttype">CANOPY</span>
+        <img src="images/dot.png" alt="dot" />
+        <span class="texttype2">Back End Dev</span>
+        <img src="images/dot.png" alt="dot" />
+        <span class="texttype2">2015</span>
+      </div>
+        <div class="img-div-modal">
+      <img class="img-hero" src="${project.image}" alt="image-project" />
+        </div>
+      <div class="project-info">
+        <div class="column1">
+          <div class="description">
+            <p class="text1-modal">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.</p>
+          </div>
+        </div>
+        <div class="column2">
+          <div class="buttons">
+            <ul class="wbtn1">
+              <li class="wbtn3">${project.html}</li>
+              <li class="wbtn3">${project.css}</li>
+              <li class="wbtn3">${project.javascript}</li>
+            </ul>
+            <hr>
+            <div class="button-container">
+              <button class="btn-modal">See sive<img class="icon" src="images/iconlive.png" alt="github" /></button>
+              <button class="btn-modal">See source<img class="icon" src="images/icongithub.png" alt="github" /></button>
+            </div>
+          </div>
+        </div>
+      </div>
+      <div><p>.</p></div>
+    </div>
+  `;
+
+  const closeButton = modal.querySelector('.close');
+  closeButton.addEventListener('click', () => {
+    modal.remove();
+  });
+
+  document.body.appendChild(modal);
+}
```

Comment:
Nice work so far👍. but let's try to stick to the given [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39-122&t=vMefwtpDudAjVPZG-0) as requested by the previous reviewer👍 currently, your popup is looking a bit different from the given design on mobile🥲 and we are required to make the popup responsive😉 Kindly take note of the comparison below and address the overflow issue and ensure your popup matches the given design on mobile👍👍

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img width="320" alt="mobile popup" src="https://user-images.githubusercontent.com/64643995/231713307-0cdf03bf-e40e-49dd-8f47-99611bda9241.png">|<img width="320" alt="design" src="https://user-images.githubusercontent.com/64643995/231713418-0e0f0b80-8263-45e9-9b4d-233348738179.png">|